index.html

```diff
@@ -1414,8 +1447,43 @@ <h3>Selective Disclosure</h3>
       <section>
         <h3>Predicates</h3>
         <p>
-Predicates allow a <a>holder</a> to provide True/False values to a
-<a>verifier</a> rather than revealing claim values.
+Another technique which may be used to support progressive trust is to use
+predicates as the values of revealed claims. Predicates allow a <a>holder</a> to
+provide True/False values to a <a>verifier</a> rather than revealing claim
+values.
+        </p>
+        <p>
+Predicate proofs may be enabled by <a>verifiable credential</a> <a>issuers</a>
+as claims, e.g., the <a>credentialSubject</a> may include a
```

Comment:
`include a` 
-> `include an`

Reply:
addressed in [a2642eb](https://github.com/w3c/vc-imp-guide/pull/14/commits/a2642eb58336bcc4303270b72f9e3ffcc8cd74bc)